src/main/java/uk/gov/beis/els/api/categories/tumbledryers/TumbleDryerInternetLabelApiForm.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.beis.els.api.categories.tumbledryers;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import javax.validation.constraints.NotNull;
+import uk.gov.beis.els.api.common.ApiValuesFromLegislationCategory;
+import uk.gov.beis.els.api.common.BaseInternetLabelApiForm;
+import uk.gov.beis.els.categories.tumbledryers.service.TumbleDryersService;
+
+@Schema(name = "Tumble dryer arrow image")
+public class TumbleDryerInternetLabelApiForm  extends BaseInternetLabelApiForm {
+
+  @Schema(description = "The energy efficiency class indicator")
```

Comment:
can use "Energy efficiency class of the tumble dryer" to keep consistency with the internet form

Reply:
Have we been doing that elsewhere? I thought we left it generic on purpose in the end?

Reply:
https://github.com/UKGovernmentBEIS/energy-label-service/pull/92#discussion_r803496696
I thought generic when there is more than one variation or same as the html form when it was the same on all.

Reply:
I think having it generic is fine. We can review the wording as part of the documentation task.